shared/locales/phone_types/phone_types.en-US.l20n

```diff
@@ -0,0 +1,25 @@
+/* Field labels */
+<phone_type {
+  *mobile:    "Mobile",
+  home:       "Home",
+  work:       "Work",
+  personal:   "Personal",
+  fax_home:   "Fax home",
+  fax_office: "Fax office",
+  fax_other:  "Fax other",
+  other:      "Other"
+}>
+<_phone_type[$type] {
+  *mobile:    "{{ phone_type.mobile }}",
+  home:      "{{ phone_type.home }}",
+  work:      "{{ phone_type.work }}",
+  personal:  "{{ phone_type.personal }}",
+  faxHome:   "{{ phone_type.fax_home }}",
+  faxOffice: "{{ phone_type.fax_office }}",
+  faxOther:  "{{ phone_type.fax_other }}",
+  other:     "{{ phone_type.other }}"
+}>
+<phone_type_and_carrier "{{ _phone_type }}, {{ carrier }}">
+<custom_phone_type_and_carrier "{{ $type }}, {{ carrier }}">
+<custom_phone_type "{{ $type }}">
```

Comment:
When a custom phone type is given we should not localize it but just print it out as-is. carrier should have been $carrier obviously.